include/revng/Lift/CSVOffsets.h

```diff
@@ -6,6 +6,8 @@
 
 #include <set>
 
+#include "revng/Support/Assert.h"
+
```

Comment:
Uh=

Reply:
not sure what the issue is about this line? there is a use of revng_assert in the file